documentation/book/cluster-operator.adoc

```diff
@@ -96,22 +96,22 @@ An object allowing the JVM running Kafka to be configured.
 The accepted format is described in the <<jvm_json_config>> section.
 `spec.kafka.rack`::
 An object allowing the Kafka rack feature to be configured and used in rack-aware partition assignment for fault tolerance.
-The accepted JSON format is described in the <<kafka_rack>> section.
+For information about the accepted JSON format, see <<kafka_rack>> section.
```

Comment:
You don't need to change this now, but going forward, the proper style for internal links is to use the xref notation as here: http://strimzi.io/contributing/guide/#styling_links